source/sync/data-model/create-a-schema.txt

```diff
@@ -16,53 +21,72 @@ Create a Data Model
 Overview
 --------
 
-You can create a schema for your App in one of two ways:
+The Device Sync data model consists of two consistent schemas: the Realm Object Schema and the App Services Schema. 
```

Comment:
I assume I included "Atlas" there bc it was the first instance. But if we're going for consistency, I'd prefer just "App Services Schema" instead of including Atlas every time, if only to keep it _slightly_ shorter.